test/ranking.spec.js

```diff
@@ -0,0 +1,112 @@
+import {
+    Ranking
+} from '../src/app/Ranking';
+
+describe("Ranking's logic", () => {
+    const peopleRanking = new Ranking('people');
+
+    const results = [{
+        'user': 'user1',
+        'score': 15,
+        'maxScore': 30 // 50%
+    }, {
+        'user': 'user2',
+        'score': 18,
+        'maxScore': 40 // 45%
+    }, {
+        'user': 'user3',
+        'score': 15,
+        'maxScore': 25 // 60%
+    }];
+
+    afterEach(() => {
+        localStorage.clear();
+    });
+
+    it('When set wrong category name, exception will be thrown', () => {
+        expect(() => {
+            new Ranking('vehicle');
+        }).toThrow('"vehicle" mode is not valid!');
+    });
+
+    it('When new score is higher than or equal to at least one of top results, return true', () => {
+        localStorage.setItem('people', JSON.stringify(results));
+        const score = 9;
+        const maxScore = 18; // 45%
+
+        expect(peopleRanking._isHigherThanCurrentTop(score, maxScore)).toBeTruthy;
+    });
+
+    it('When new score is less than all of top results, return false', () => {
+        localStorage.setItem('people', JSON.stringify(results));
+        const score = 11;
+        const maxScore = 25; // 44%
+
+        expect(peopleRanking._isHigherThanCurrentTop(score, maxScore)).toBeFalsy;
+    });
+
+    it('When there are three results, return two best of them.', () => {
+        localStorage.setItem('people', JSON.stringify(results));
+        const bestResults = [{
+            'user': 'user1',
+            'score': 15,
+            'maxScore': 30 // 50%
+        }, {
+            'user': 'user3',
+            'score': 15,
+            'maxScore': 25 // 60%
+        }];
+
+        expect(peopleRanking._getTwoHighestResults()).toEqual(bestResults);
+    });
+
+    it('When less than three results, add a new result to the top scores', () => {
+        const savedResults = [{
```

Comment:
Dobrze byłoby nie mieszać setupu testów (opisu warunków początkowych) z asercją. 
Czyli te savedResults wykorzystujesz tylko w asercji to dobrze przenieść je niżej.

Reply:
Niżej, to znaczy na sam koniec testu czy w którym miejscu?

Reply:
Poprawiłam, o to chodziło?

Reply:
Moze byc i tak, chociaz ja chyba w ogole bym po prostu wrzucil te wartosci w miejsce gdzie sa nazwy. Bo uzywamy tego w jednym mijescu :)